pydantic/main.py

```diff
@@ -761,6 +764,14 @@ def __getattr__(self, item: str) -> Any:
                         # this is the current error
                         raise AttributeError(f'{type(self).__name__!r} object has no attribute {item!r}')
 
+        def __getattribute__(self, item: str) -> Any:
+            try:
+                msg = type(self).__pydantic_deprecated_messages__[item]
+                warnings.warn(msg, builtins.DeprecationWarning, stacklevel=2)
```

Comment:
`builtins` have to be used as `DeprecationWarning` is set to `PydanticDeprecatedSince20` line `#55`